DFE-Digital/teaching-vacancies
# Update tests for jobseeker view of job show page

Previously, the spec only tested the following scenario:
1. Load the job show page
2. Become logged in
3. Click a button etc etc

We also need to test the more common scenario:
1. Be logged in already
2. Load the job show page when logged in
3. Click a button etc etc

This lets us test, among other things, that the correct CTA is displayed on the banner button, depending on the job application status (draft etc).

We should continue to test the 'load page first' scenario, because we want to display specific flash messages in these cases.
